llvm/include/llvm/Support/Error.h

```diff
@@ -493,11 +493,12 @@ template <class T> class [[nodiscard]] Expected {
 
 public:
   /// Create an Expected<T> error value from the given Error.
-  Expected(Error Err)
+  Expected(Error &&Err)
       : HasError(true)
 #if LLVM_ENABLE_ABI_BREAKING_CHECKS
         // Expected is unchecked upon construction in Debug builds.
-        , Unchecked(true)
+        ,
```

Comment:
nit: revert this formatting change?

Reply:
I thought I did - in any case, it may upset the clang-format CI, but I agree formatting should happen in a separate patch.